{DiffBn_coarse: 4, DiffBn_fine: 39, OFFBn_coarse: 4, OFFBn_fine: 0, ONBn_coarse: 6,
  ONBn_fine: 200, PrBp_coarse: 2, PrBp_fine: 58, PrSFBp_coarse: 1, PrSFBp_fine: 33,
  RefrBp_coarse: 4, RefrBp_fine: 25, aps_enabled: true, dvs_enabled: true, exposure: 4000,
  frame_delay: 0, imu_acc_scale: 3, imu_enabled: true, imu_gyro_scale: 3, max_events: 0,
  streaming_rate: 30}

